### ISSUE TITLE
如何修改config.json的加载路径(how to modify the load path of config.json?) 

### ISSUE THREAD

**Issue by TianqiFu** [yangheng95/PyABSA#363]
hey yang, 

由于国内对huggingface.co的管制无法直接自动下载。我需要手动下载。所以请教您如何使用本地的这些文件（config.json）。
cause china network connot access huggingface，i need manual download config.json, so please point out how to use config.json
at location.

thx.

**Comment by TianqiFu** [comment]
o i find the way to modify,
just like this: 
config.pretrained_bert = 'D:/PythonProject/pyabsademo/test'

**Comment by TianqiFu** [comment]
version:2.3.4
o another issue happen, as below pic, when i run the program it just stop there. i have not idea to fix it. please have a look. thx 
<img width="821" alt="微信图片_20231122185741" src="https://github.com/yangheng95/PyABSA/assets/19373187/47b61a64-de7d-4061-ad9c-061786ee2dd4">

**Comment by yangheng95** [comment]
This problem has not been reported. Try to disable the load_aug=False

**Comment by TianqiFu** [comment]
> This problem has not been reported. Try to disable the load_aug=False

i have tried this option， but it seem didn't work. i wander that it's may caused by env packages.

when you free could you please check the env package list as below(attachment is 
[env packages.txt](https://github.com/yangheng95/PyABSA/files/13454512/env.packages.txt)
 .txt of the list
):

# packages in environment at D:\software\anaconda3\envs\pytorchGPU:
#
# Name                    Version                   Build  Channel
annotated-types           0.6.0                    pypi_0    pypi
autocuda                  0.16                     pypi_0    pypi
blis                      0.7.11                   pypi_0    pypi
boostaug                  2.3.5                    pypi_0    pypi
brotli-python             1.1.0            py38hd3f51b4_1    conda-forge
ca-certificates           2023.11.17           h56e8100_0    conda-forge
catalogue                 2.0.10           py38haa244fe_0    conda-forge
certifi                   2022.12.7                pypi_0    pypi
charset-normalizer        2.0.12                   pypi_0    pypi
click                     8.1.7           win_pyh7428d3b_0    conda-forge
cloudpathlib              0.16.0                   pypi_0    pypi
colorama                  0.4.6              pyhd8ed1ab_0    conda-forge
commonmark                0.9.1                      py_0    conda-forge
confection                0.1.3            py38h009e608_1    conda-forge
contourpy                 1.1.1                    pypi_0    pypi
cycler                    0.12.1                   pypi_0    pypi
cymem                     2.0.8            py38hd3f51b4_1    conda-forge
cython-blis               0.7.10           py38he7056a7_2    conda-forge
dataclasses               0.8                pyhc8e2a94_3    conda-forge
et-xmlfile                1.1.0                    pypi_0    pypi
filelock                  3.9.0                    pypi_0    pypi
findfile                  2.0.1                    pypi_0    pypi
fonttools                 4.44.0                   pypi_0    pypi
fsspec                    2023.10.0                pypi_0    pypi
future                    0.18.3             pyhd8ed1ab_0    conda-forge
gitdb                     4.0.11                   pypi_0    pypi
gitpython                 3.1.40                   pypi_0    pypi
huggingface-hub           0.17.3                   pypi_0    pypi
idna                      3.4                pyhd8ed1ab_0    conda-forge
importlib-resources       6.1.1                    pypi_0    pypi
intel-openmp              2023.2.0         h57928b3_50497    conda-forge
ipywidgets                8.1.1                    pypi_0    pypi
jinja2                    3.1.2              pyhd8ed1ab_1    conda-forge
joblib                    1.3.2                    pypi_0    pypi
jupyterlab-widgets        3.0.9                    pypi_0    pypi
kiwisolver                1.4.5                    pypi_0    pypi
langcodes                 3.3.0              pyhd8ed1ab_0    conda-forge
libblas                   3.9.0              19_win64_mkl    conda-forge
libcblas                  3.9.0              19_win64_mkl    conda-forge
libhwloc                  2.9.3           default_haede6df_1009    conda-forge
libiconv                  1.17                 h8ffe710_0    conda-forge
liblapack                 3.9.0              19_win64_mkl    conda-forge
libxml2                   2.11.6               hc3477c8_0    conda-forge
libzlib                   1.2.13               hcfcfb64_5    conda-forge
markupsafe                2.1.2                    pypi_0    pypi
matplotlib                3.7.3                    pypi_0    pypi
metric-visualizer         0.9.9                    pypi_0    pypi
mkl                       2023.2.0         h6a75c08_50496    conda-forge
mpmath                    1.3.0                    pypi_0    pypi
murmurhash                1.0.10           py38hd3f51b4_1    conda-forge
natsort                   8.4.0                    pypi_0    pypi
networkx                  3.0                      pypi_0    pypi
numpy                     1.24.1                   pypi_0    pypi
openpyxl                  3.1.2                    pypi_0    pypi
openssl                   1.1.1w               hcfcfb64_0    conda-forge
packaging                 23.2               pyhd8ed1ab_0    conda-forge
pandas                    2.0.3                    pypi_0    pypi
pathy                     0.10.2             pyhd8ed1ab_0    conda-forge
pillow                    9.3.0                    pypi_0    pypi
pip                       23.3             py38haa95532_0
preshed                   3.0.9            py38hd3f51b4_1    conda-forge
protobuf                  3.20.3                   pypi_0    pypi
pthreads-win32            2.9.1                hfa6e2cd_3    conda-forge
pyabsa                    2.3.4                    pypi_0    pypi
pydantic                  2.4.2                    pypi_0    pypi
pydantic-core             2.10.1                   pypi_0    pypi
pygments                  2.17.1             pyhd8ed1ab_0    conda-forge
pyparsing                 3.1.1                    pypi_0    pypi
pysocks                   1.7.1              pyh0701188_6    conda-forge
python                    3.8.0                hff0d562_2
python_abi                3.8                      2_cp38    conda-forge
pytorch-warmup            0.1.1                    pypi_0    pypi
pytz                      2023.3.post1             pypi_0    pypi
pyyaml                    6.0.1                    pypi_0    pypi
regex                     2023.10.3                pypi_0    pypi
requests                  2.27.1                   pypi_0    pypi
rich                      12.6.0             pyhd8ed1ab_0    conda-forge
safetensors               0.4.0                    pypi_0    pypi
scikit-learn              1.2.2                    pypi_0    pypi
scipy                     1.10.1                   pypi_0    pypi
sentencepiece             0.1.99                   pypi_0    pypi
seqeval                   1.2.2                    pypi_0    pypi
setuptools                68.0.0           py38haa95532_0
shellingham               1.5.4              pyhd8ed1ab_0    conda-forge
smart-open                6.4.0                    pypi_0    pypi
smart_open                5.2.1              pyhd8ed1ab_0    conda-forge
smmap                     5.0.1                    pypi_0    pypi
spacy                     3.7.2                    pypi_0    pypi
spacy-legacy              3.0.12             pyhd8ed1ab_0    conda-forge
spacy-loggers             1.0.5              pyhd8ed1ab_0    conda-forge
spacy-model-en_core_web_sm 3.5.0              pyhd8ed1ab_0    conda-forge
sqlite                    3.41.2               h2bbff1b_0
srsly                     2.4.8            py38hd3f51b4_1    conda-forge
sympy                     1.12                     pypi_0    pypi
tabulate                  0.9.0                    pypi_0    pypi
tbb                       2021.10.0            h91493d7_2    conda-forge
termcolor                 2.3.0                    pypi_0    pypi
thinc                     8.2.1                    pypi_0    pypi
threadpoolctl             3.2.0                    pypi_0    pypi
tikzplotlib               0.10.1                   pypi_0    pypi
tokenizers                0.14.1                   pypi_0    pypi
torch                     2.1.0+cu121              pypi_0    pypi
torchaudio                2.1.0+cu121              pypi_0    pypi
torchvision               0.16.0+cu121             pypi_0    pypi
tqdm                      4.66.1             pyhd8ed1ab_0    conda-forge
transformers              4.35.0                   pypi_0    pypi
typer                     0.9.0                    pypi_0    pypi
typing-extensions         4.8.0                hd8ed1ab_0    conda-forge
typing_extensions         4.8.0              pyha770c72_0    conda-forge
tzdata                    2023.3                   pypi_0    pypi
ucrt                      10.0.22621.0         h57928b3_0    conda-forge
update-checker            0.18.0                   pypi_0    pypi
urllib3                   1.25.11                  pypi_0    pypi
vc                        14.2                 h21ff451_1
vc14_runtime              14.36.32532         hdcecf7f_17    conda-forge
vs2015_runtime            14.36.32532         h05e6639_17    conda-forge
wasabi                    1.1.2            py38haa244fe_0    conda-forge
weasel                    0.3.4                    pypi_0    pypi
webcolors                 1.13                     pypi_0    pypi
wheel                     0.41.2           py38haa95532_0
widgetsnbextension        4.0.9                    pypi_0    pypi
win_inet_pton             1.1.0              pyhd8ed1ab_6    conda-forge
xlsxwriter                3.1.9                    pypi_0    pypi

**Comment by yangheng95** [comment]
你能够在pycharm或者其他工具中通过单步调试找到卡住的地方吗，不然不好定位问题

**Comment by TianqiFu** [comment]
我降低了版本到2.0.0，可以运行，
